runner/app/pipelines/audio_to_text.py

```diff
@@ -65,6 +66,17 @@ def __init__(self, model_id: str):
 
         torch_device = get_torch_device()
 
+        # Get the GPU device properties
+        device = torch.cuda.get_device_properties(0)
+        major, minor = device.major, device.minor
+
+        # Compute capability of Ampere starts at 8.0
+        if (major, minor) >= (8, 0):
+            attn_implementation = "flash_attention_2"
+        else:
+            warnings.warn(f"GPU {device.name} (Compute Capability {major}.{minor}) is not compatible with FlashAttention so, scaled_dot_product_attention is being used instead")
```

Comment:
@RUFFY-369 is there any reason to use the warnings package over logger.warn?

Reply:
@rickstaa Not a specific reason but just followed basic code habit that I have for using warnings package for issuing warning which devs can find useful as logger.warn will log it for the user but it isn't that critical for a log so left it out just as a warning display with warnings package.

If you want I can add this warning as a log :+1:

Reply:
I'll check it tomorrow. It's a good habit, and I suggested it mainly to maintain consistency in the code.

Reply:
Sure, then let me just log in the warning as it would be more in line with the code pattern of all the other pipeline files too. :+1: